BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/html_file/views/_html.html.erb

```diff
@@ -0,0 +1,56 @@
+
+<%
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2023
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+%>
+
+
+<%= link_to(
```

Comment:
Can you please restructure this the 'standard' way? e.g.

```erb
  <% if page_is?('#show') %>
    <%= link_to ... %>
  <% end %>
```

The reason is, your big `<%=` block is reduced to nil of the condition is false, but we have to read all the way until the end to realize that it can print nothing.